marklogic/manifest.json

```
@@ -0,0 +1,34 @@
+{
+  "display_name": "MarkLogic",
+  "maintainer": "help@datadoghq.com",
+  "manifest_version": "1.0.0",
+  "name": "marklogic",
+  "metric_prefix": "marklogic.",
+  "metric_to_check": "marklogic.hosts.memory_size",
```

(nit)

```suggestion
  "metric_to_check": "marklogic.hosts.total_hosts",
```

`marklogic.hosts.total_hosts` seems more natural to use as `metric_to_check`.